core/modules/system/system.install

```diff
@@ -219,10 +219,7 @@ function system_requirements($phase) {
 
     if (!empty($description)) {
       if ($php_ini_path = get_cfg_var('cfg_file_path')) {
-        $description .= ' ' . $t('Increase the memory limit by editing the memory_limit parameter in the file %configuration-file and then restart your web server (or contact your system administrator or hosting provider for assistance).', array('%configuration-file' => $php_ini_path));
-      }
-      else {
-        $description .= ' ' . $t('Contact your system administrator or hosting provider for assistance with increasing your PHP memory limit.');
+        $description .= ' ' . $t('Increase the memory limit by editing the memory_limit parameter in the file %configuration-file and then restart your web server.', array('%configuration-file' => $php_ini_path));
```

Comment:
What if they can't? I don't think many shared webhosting providers allow any user to restart the web server. :wink:

Reply:
I don't know. I just know that this text was always there, I just removed the 'contact an admin' parts...

Reply:
> I just know that this text was always there

Sure, but I see potential for improvement. The text has probably been untouched for a while, because hardly anyone ever sees it.
Anyway, your decision. :wink:

Reply:
There are a number of error messages where we could provide suggestions on how to resolve it.  Adding a link to a page on b.o that goes into detail about updating your php.ini and what to do for common hosting situations -- Ubuntu vs RHEL vs cheap shared hosting vs pro-grade shared hosting.)  That said, such changes are well beyond the scope of this issue.

Reply:
Yes, I'd say a new issue should be opened for this.